framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -203,15 +213,38 @@ public Void visitDeclared(AnnotatedDeclaredType type, Void p) {
          * If any constant-value annotation has &gt; MAX_VALUES number of values provided, treats
          * the value as UnknownVal. Works together with ValueVisitor.visitAnnotation, which issues a
          * warning to the user in this case.
+         *
+         * <p>Additional rules applied for IntVal and IntRange annotations: if anno is IntVal and
+         * the number of values provided is &gt; MAX_VALUES, replace with IntRange; if anno is
+         * IntRange and the range is &le; MAX_VALUES, replace with IntVal
```

Comment:
End sentences with punctuation (here, a period).
Why is the annotation replaced with IntVal?  (That is, what goal does that implementation choice serve?)